### PR TITLE
Update obt version

### PR DESCRIPTION
**Pull request by andygout**
cc @keirog

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,7 @@ tidy:
 	nbt destroy ${TEST_HOST}
 
 install:
+	npm install bower@1.6.5
```
Comment (matthew-andrews) [inline Makefile]: how come?

Comment (matthew-andrews) [inline Makefile]: if we need to do this can we do this int he package.json rather than makefile?

**Comment by keirog** [comment]
See [comment in next-opt-in-out](https://github.com/Financial-Times/next-opt-in-out/pull/91#issuecomment-159354236)--it may be the same here?

**Comment by andygout** [comment]
Agreed that upgrading node to v4.2.2 is the best solution ( @keirog 's branch below ) so am closing this branch:

https://github.com/Financial-Times/next-errors/tree/bump-node

**Pull request closed without merging**